examples/source/1.components/amp-script/index.html

```diff
@@ -158,6 +173,36 @@
         fetchCurrentTime();
       </script>
     </div>
+
+    <!--
+      ## Using a WebSocket for a live blog
```

Comment:
Suggestion: make the title more generic (e.g. Live updates via sockets). Due to the resizing problem, live blogs are tricky with amp-script and I'm not sure that we should recommend this.

Reply:
Changed title.

Reply:
The updated title has the same problem.